app/assets/javascripts/discourse/app/lib/d-tooltip.js

```diff
@@ -0,0 +1,22 @@
+import tippy from "tippy.js";
+
+function stopPropagation(instance, event) {
+  event.preventDefault();
+  event.stopPropagation();
+}
+function hasTouchCapabilities() {
+  return navigator.maxTouchPoints > 0;
```

Comment:
are you sure about this?

Our capabilities.js service defines this like this:

```javascript
this.touch = navigator.maxTouchPoints > 1 || "ontouchstart" in window;
```

Reply:
it always worked well for me, but maybe the above it's more reliable
I'm down to change it, even if it's just for consistency